packages/form-js-editor/test/spec/FormEditor.spec.js

```diff
@@ -468,22 +455,22 @@ describe('FormEditor', function() {
       it('should update on selection changed', async function() {
 
         // given
-        const form = await createFormEditor({
+        const formEditor = await createFormEditor({
           container,
           schema
         });
 
-        // when
-        form.get('selection').set('text');
+        await waitForSelected('Form_1');
 
-        await waitFor(async () => {
-          const input = await screen.getByLabelText('Text');
+        const text1 = formEditor.get('formFieldRegistry').get('Text_1');
 
-          expect(input).to.exist;
-        });
+        // when
+        formEditor.get('selection').set(text1);
+
+        await waitForSelected('Text_1');
 
         // then
-        expectSelected('text');
+        expectSelected('Text_1');

```

Comment:
Feels redundant, kind-of.

Why not alias "waitForSelected" with `expectSelected`:

```
// then
await expectSelected('SomeFieldId');
```

Reply:
Yeah, combining the two makes sense. 👍🏻